docs/index.md

```diff
@@ -167,9 +167,141 @@ Returning:
 </div>
 ```
 
-#### Slots (experimental)
+#### Slots
 
-_Slots are currently under development as a successor to Content Areas. The Slot APIs should be considered unfinished and subject to breaking changes in non-major releases of ViewComponent._
+_Slots now refer to V2 of the slot implementation, which is the successor to Slots V1 and Content Areas._
+
+Slots enable multiple blocks of content to be passed to a single ViewComponent.
+
+Slots expose two helper methods for defining slots, `renders_one` and
+`renders_many`.
+
+`renders_one` allows you to define a slot that will be rendered at most once
+per-page. e.g. `renders_one :header`
```

Comment:
> at most once per-page

Just for clarity, I *could* render it twice if I wanted to? Or is it technically really not possible/intended? 

Or did you mean this?

```suggestion
per component. e.g. `renders_one :header`
```

Reply:
I definitely meant per-component 😄 

Although re-reading it, I think saying `can be defined at most once per-component render` might be more accurate/correct?